modules/chat/client/controllers/chat.client.controller.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 // Create the 'chat' controller
-angular.module('chat').controller('ChatController', ['$scope', 'Socket',
-    function($scope, Socket) {
+angular.module('chat').controller('ChatController', ['$scope', '$location', 'Authentication', 'Socket',
```

Comment:
@rhutchison mentioned we probably want to refactor that in a larger scale so definitely after 0.4.0 but would be good to have couple more eyes on this PR before we merge.